config/development.json

```diff
@@ -177,7 +177,7 @@
 		"stats/horizontal-bars-everywhere": true,
 		"stats/insights-page-grid": true,
 		"stats/new-video-summary": true,
-		"stats/subscribers-section": false,
+		"stats/subscribers-section": true,
```

Comment:
Let's refrain from doing this since the subscriber section isn't ready to be shown to anyone outside our direct contributors.